src/valkey-cli.c

```diff
@@ -3091,17 +3091,17 @@ version,tls_usage);
 "                     Use filename of \"-\" to write to stdout.\n"
 "  --functions-rdb <filename> Like --rdb but only get the functions (not the keys)\n"
 "                     when getting the RDB dump file.\n"
-"  --pipe             Transfer raw Redis protocol from stdin to server.\n"
+"  --pipe             Transfer raw Valkey protocol from stdin to server.\n"
```

Comment:
The protocol is called RESP. Let's use this word when talking about the protocol.

```suggestion
"  --pipe             Transfer raw RESP protocol from stdin to server.\n"
```

Reply:
updated accordingly. thanks!